test/core/vm/actor/CMakeLists.txt

```diff
@@ -11,3 +11,13 @@ target_link_libraries(actor_test
     cbor
     hexutil
     )
+
+addtest(init_actor_test
+    init_actor_test.cpp
+    )
+target_link_libraries(init_actor_test
+    actor
```

Comment:
test's target_link_libraries section should be as small as possible, ideally containing one library - tested one and probably testutils. if it is not compiled you probably should link them to the original library.